src/chessboard/types/index.ts

```diff
@@ -356,4 +356,10 @@ export type ChessboardProps = {
    * @default false
    */
   autoPromoteToQueen?: boolean;
+
+  dragOutsideOfBoard?: boolean;
```

Comment:
Can we change this to `allowDragOutsideBoard`

Reply:
can the new prop also be added to the readme, in the correct alphabetical order placement please

Reply:
Moved new prop and autoPromotToQueen to correct position